common/supplemental/likelySubtags.xml

```diff
@@ -3014,8 +3014,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; ?; Spain } => { Spanish; Latin; Spain }-->
 		<likelySubtag from="und_ET" to="am_Ethi_ET"/>
 		<!--{ ?; ?; Ethiopia } => { Amharic; Ethiopic; Ethiopia }-->
-		<likelySubtag from="und_EU" to="en_Latn_GB"/>
-		<!--{ ?; ?; European Union } => { English; Latin; United Kingdom }-->
+		<likelySubtag from="und_EU" to="en_Latn_IE"/>
+		<!--{ ?; ?; European Union } => { English; Latin; Ireland }-->
```

Comment:
Since this is not coming from country_language_population_raw.txt, isn't there an override we have to add somewhere to keep this from getting overwritten during the BRS process?

Reply:
I made those changes, and reran the tools.

supplementalMetaData - unchanged

likelySubtags - changed (but not doi)

		<likelySubtag from="hi_Latn" to="hi_Latn_IN"/>
		<!--{ Hindi; Latin; ? } => { Hindi; Latin; India }-->
Redundant, since 		<likelySubtag from="hi" to="hi_Deva_IN"/>

		<likelySubtag from="ks_Deva" to="ks_Deva_IN"/>
		<!--{ Kashmiri; Devanagari; ? } => { Kashmiri; Devanagari; India }-->
Redundant, since 		<likelySubtag from="ks" to="ks_Arab_IN"/>

		<likelySubtag from="ms_ID" to="ms_Latn_ID"/>
		<!--{ Malay; ?; Indonesia } => { Malay; Latin; Indonesia }-->
Redundant, since 		<likelySubtag from="ms" to="ms_Latn_MY"/>

Will have to do some research on ks.